README.md

```diff
@@ -45,7 +45,25 @@ The UI begins as a blank slate -- you can populate it with plots, images, and te
 
 > **Tip**: You can use the zoom of your browser to adjust the scale of the UI.
 
+##### Callbacks
 
+The python Visdom implementation supports callbacks on a pane. The demo shows an example of this in the form of an editable text pad. The functionality of these callbacks allows the Visdom object to recieve and react to events that happen in the frontend.
+
+You can subscribe a pane to events by adding a function to the `viz.event_handlers` dict for the pane id you want to subscribe. When an event occurs to that pane, your callback will be called on a dict containing:
+
+ - `eventType`: one of the below event types
+ - `paneData`: all of the stored contents for that pane including layout and content.
+ - `eid`: the current environment id
+ - `target`: the pane id the event is called on
+
+Additional parameters are defined below.
+
+Right now two callback events are supported:
+
+1. `close` - Triggers when a window is closed. Returns a dict with only the aforementioned fields.
+2. `keyPress` - Triggers when a key is pressed. Contains additional parameters:
+    - `key` - A string representation of the key pressed (applying state modifiers such as SHIFT)
+    - `keyCode` - The javascript event keycode for the pressed key (no modifiers)
```

Comment:
To be consistent with the other input arguments in visdom, I think it is better to not use camel-case argument names here.

Reply:
Makes sense - I wasn't really sure where to draw the line between our JS components (all camel case) and the python stuff

Reply:
I see. Most users will likely be accessing these variables from Python, so for them, it makes sense to use lower-case. You could "translate" the variable names in `check_origin()` if you want to keep camel-case in the JS code (admittedly, that is not very elegant though).

Reply:
JavaScript in nature isn't elegant, I'll leave the mismatched naming there 😆